### PR TITLE
Added new Portuguese language header file

### PR DESCRIPTION
**Pull request by AndreTeixeira1998**

### CODE REVIEW

File: src/lang/pt_PT.h [hunk 1]
```diff
@@ -0,0 +1,205 @@
+#ifndef HASP_LANG_PT_PT_H
+#define HASP_LANG_PT_PT_H
+
+#define D_USERNAME "Usuário:"
```
Comment (dgomes) [inline src/lang/pt_PT.h:4]: Usuário is PT_br
Utilizador is PT_pt

File: src/lang/pt_PT.h [hunk 2]
```diff
@@ -0,0 +1,205 @@
+#ifndef HASP_LANG_PT_PT_H
+#define HASP_LANG_PT_PT_H
+
+#define D_USERNAME "Usuário:"
+#define D_PASSWORD "Senha:"
```
Comment (dgomes) [inline src/lang/pt_PT.h:5]: Palavra-passe

File: src/lang/pt_PT.h [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+#define D_SERVICE_CONNECTED "Conectado"
+#define D_SERVICE_DISCONNECTED "Desconectado"
+
+#define D_SETTING_ENABLED "Ativado"   // New
+#define D_SETTING_DISABLED "Desativado" // New
+
+#define D_NETWORK_IP_ADDRESS_RECEIVED "Foi recebido no endereço IP: %s"
+#define D_NETWORK_ONLINE "Online"
+#define D_NETWORK_OFFLINE "Offline"
+#define D_NETWORK_CONNECTION_FAILED "Falhou a conexão"
```
Comment (dgomes) [inline src/lang/pt_PT.h:40]: ligação

**Comment by fvanroie** [comment]
Thank you very much for this contribution!

File: src/lang/pt_PT.h [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+#define D_HTTP_SENDING_PAGE "Foi enviado página %S a %s" // New
+#define D_HTTP_FOOTER "por Francis Van Roie"
+
+#define D_INFO_VERSION "Versão"
+#define D_INFO_BUILD_DATETIME "Data de compilação"
+#define D_INFO_UPTIME "Tempo ativo"
+#define D_INFO_FREE_HEAP "Heap livre"
+#define D_INFO_FREE_BLOCK "Blocos livres"
+#define D_INFO_DEVICE_MEMORY "Memória do dispositivo"
+#define D_INFO_LVGL_MEMORY "Memoria LVGL"
```
Comment (dgomes) [inline src/lang/pt_PT.h:144]:
```suggestion
#define D_INFO_LVGL_MEMORY "Memória LVGL"

```

File: src/lang/pt_PT.h [hunk 5]
```diff
@@ -0,0 +1,205 @@
...
+#define D_HTTP_CALIBRATE "Calibrar"
+#define D_HTTP_SCREENSHOT "Screenshot"
+#define D_HTTP_FILE_BROWSER "Editor de ficheiros"
+#define D_HTTP_FIRMWARE_UPGRADE "Atualização do firmware"
+#define D_HTTP_UPDATE_FIRMWARE "Atualizar o firmware"
+#define D_HTTP_FACTORY_RESET "Restauração de fábrica"
+#define D_HTTP_MAIN_MENU "Menu Principal"
+#define D_HTTP_REBOOT "Reiniciar"
+#define D_HTTP_CONFIGURATION "Configuração"
+#define D_HTTP_SENDING_PAGE "Foi enviado página %S a %s" // New
```
Comment (dgomes) [inline src/lang/pt_PT.h:135]:
```suggestion
#define D_HTTP_SENDING_PAGE "Foi enviado página %s a %s" // New

```

**Review by dgomes — CHANGES_REQUESTED** [review]
**Conetado** é uma tradução muito literal, proponho **ligado**

File: src/lang/pt_PT.h (hunk 5 above)
Comment (fvanroie) [reply src/lang/pt_PT.h:135]: There is actually one `%S` in the file. I need to verify if this is still correct, but `%S` should be fine in this line.

**Comment by fvanroie** [comment]
Thanks for reviewing @dgomes .

File: src/lang/pt_PT.h (hunk 3 above)
Comment (AndreTeixeira1998) [reply src/lang/pt_PT.h:40]: Yes, it is

File: src/lang/pt_PT.h (hunk 4 above)
Comment (fvanroie) [reply src/lang/pt_PT.h:144]: Updated it

**Pull request merged**